docs-ref-services/latest/app-configuration-readme.md

```diff
@@ -1,19 +1,17 @@
 ---
-title: 
+title: Use App Configuration client library for JavaScript
 keywords: Azure, javascript, SDK, API, @azure/app-configuration, 
 author: maggiepint
 ms.author: magpint
 ms.date: 07/07/2020
 ms.topic: article
-ms.prod: azure
-ms.technology: azure
 ms.devlang: javascript
-ms.service: 
+ms.service: azure
```

Comment:
Shouldnt this be `azure-app-configuration`? Similarly other pages are for specific Azure services.

Reply:
This is update through our scripts. It will be overwritten by next release if change manually.

Reply:
@sima-zhu Are all the attributes here updated by automation whenever we release an update to the package? Regardless, we will need to update our script to use the right value for "ms.service"